### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by westy92**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,19 +37,18 @@
     "node": ">=0.10.0"
   },
   "devDependencies": {
-    "async": "^0.2.10",
-    "chai": "^1.9.1",
-    "grunt": "^0.4.4",
+    "async": "^2.0.1",
```
Comment (aoberoi) [inline package.json]: it turns out the "async" package is unused, so we can just get rid of this.

Comment (westy92) [inline package.json]: Removed.

**Comment by aoberoi** [comment]
@jeffswartz i intend to merge this PR, but a heads-up to you about the documentation generation: updating the "grunt-jsdoc" package will most likely change the output in a meaningful way.

**Comment by aoberoi** [comment]
thanks for helping tiding this up @westy92! you're super fast and awesome 😄

**Comment by westy92** [comment]
You're welcome! I'm glad I could contribute. :)

**Pull request merged**